src/scss/packages/image-example/src/_styles.scss

```diff
@@ -0,0 +1,26 @@
+@use 'uswds-core' as *;
+
+.image-example .bottom-border-green {
```

Comment:
I think we should name classes using semantics, not colors. Similar to how alert banner varies on [states like success & error](https://designsystem.digital.gov/design-tokens/color/state-tokens/), not green and red.

I'd expect something like `.image-example--correct` and `.image-example--incorrect` as modifier classes, rather than green and red. I'd also expect the modifier should take place at the top-level element and the child elements would have standard class names, where we can customize appearance of those child elements using CSS nesting.

Example:

```html
<div class="image-example image-example--correct">
  <!-- ... -->
  <img src="image-example__image">
</div>
```

```scss
.image-example__image {
  // ...default image styles
}

.image-example--correct .image-example__image {
  border-bottom: units(1) solid color('success');
}

// Or:
// .image-example__image {
//   // ...default image styles
//
//   .image-example--correct & {
//     border-bottom: units(1) solid color('success');
//   }
// }
```

Reference:

- https://en.bem.info/methodology/naming-convention/#two-dashes-style
- https://en.bem.info/methodology/key-concepts/

Reply:
This was really helpful, thanks for writing all this up. I rewrote ([commit](https://github.com/18F/identity-design-system/pull/369/commits/f56f6fcc1250f6f11ce26eb5c1796ba2a58aa80c)) the styling to:
- Be semantic
- Be applied at the top level.
- Follow the `block--elem` naming convention linked.
- Use the `units(...)` and `color(...)` utilities.

I extracted the shared styles into the `@mixin image-example($color)` mixin as well. I'm not sure about this. It neatly encapsulates that the only difference between `correct` and `incorrect` is one color. On the other hand, it depends more heavily on sass functionality and might be harder to read.